src/asm/rgbasm.5

```diff
@@ -1075,28 +1075,34 @@ command-line option, except when using overlays with
 .Ic DB
 defines a list of bytes that will be stored in the final image.
 Ideal for tables and text.
-Note that strings are not zero-terminated!
 .Bd -literal -offset indent
 DB 1,2,3,4,"This is a string"
 .Ed
 .Pp
+Alternatively, you can use
+.Ic DW
+to store a list of words (16-bit) or
+.Ic DL
+to store a list of double-words/longs (32-bit).
+.Pp
+Strings are handled a little specially: they first undergo charmap conversion (see
+.Sx Character maps ) ,
+then each resulting character is output individually.
+For example, under the default charmap, the following two lines are identical:
+.Bd -literal -offset indent
+DW "Hello!"
+DW "H", "e", "l", "l", "o", "!"
+.Ed
+.Pp
+If you do not want this, enclose the string in parentheses.
```

Comment:
"If you do not want this special handling", just to be clear about what the parentheses disable.